bazel_env.bzl

```diff
@@ -155,7 +156,14 @@ def _tool_impl(ctx):
         rlocation_path = _rlocation_path(ctx, ctx.executable.target)
 
         runfiles = ctx.runfiles(ctx.files.target)
-        runfiles = runfiles.merge(ctx.attr.target[0][DefaultInfo].default_runfiles)
+        runfiles = runfiles.merge_all([
```

Comment:
There is only ever a single target, the attribute only takes an array value because of the transition. Could you use `ctx.attr.target[0]` here and below and add a comment for clarity?

Reply:
I'm stupid, I looked at the code too quickly. I thought we were getting a list because `allow_files = True` . I didn't think to look at the transition.

Reply:
It's fixed, thank you.